napari/_qt/qt_resources/_icons.py

```diff
@@ -190,6 +190,17 @@ def _compile_qrc_pyqt5(qrc) -> bytes:
     return out
 
 
+def _compile_qrc_pyqt6(qrc) -> bytes:
+    """
+    We can't do that with PyQt6,
+
+    The maintainer has discontinued pyrcc for PyQt6
+
+    """
+
+    raise NotImplementedError('pyrcc discontinued on Pyqt6')
```

Comment:
yep, this is a big deal, and where #3254 also paused.  the move to superqt in that PR was a stop-gap since we didn't know whether/when qtpy would update.  The other thing it does is move away from scoped enums, which I think we should do as well (but is also an independent detail and perhaps nice to keep separate)

Reply:
Oh, yeah I missed this one as it was closed.